editor/plugins/org.fusesource.ide.wsdl2rest/.classpath

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry exported="true" kind="lib" path="lib/args4j-2.0.31.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/javaparser-core-2.5.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-collections-3.2.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/camel-spring-2.18.0.jar"/>
```

Comment:
Does it means that we are bound to a specific Camel version?

Reply:
The wsdl2rest tool is bound to a specific version, so I would suspect so.